src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -579,6 +580,7 @@ static void window_scenery_mouseup(rct_window* w, rct_widgetindex widgetIndex)
             gWindowSceneryPaintEnabled = 0;
             gWindowSceneryClusterEnabled = 0;
             gWindowSceneryEyedropperEnabled = !gWindowSceneryEyedropperEnabled;
+            scenery_remove_ghost_tool_placement();
```

Comment:
Why did you make this change? (What does it do if it's left out?)

Reply:
Before I added that change, I found the preview ghost of the scenery object currently being placed would stay on the map when switching to the scenery picker via hotkey. I suppose it wasn't needed before since you could only get the picker when mousing over the scenery window, so the ghost was already removed.

Reply:
Ah, that sounds logical.